portfolio/src/main/webapp/style.css

```diff
@@ -269,39 +281,147 @@ input[type=text], select, textarea {
   resize: vertical;
 }
 
-input[type=submit] {
-  background-color: #4CAF50;
-  color: white;
-  padding: 12px 20px;
-  border: none;
-  cursor: pointer;
-}
 
-input[type=submit]:hover {
-  background-color: #45a049;
+.container {
+  margin: auto;
+  height: 1100px;
 }
 
-/* Style the container/contact section */
-.contact-container {
-  border-radius: 5px;
-  background-color: #f2f2f2;
-  padding: 10px;
-  width: 80%;
-  height: 80vh;
-  align-items: center;
+form {
+  overflow: hidden;
+  padding: 5px;
+  width: 700px; 
   margin: auto;
-  position: relative;
+  align-content: center;
 }
 
-.contact-body h6 {
+.contact-form {
+  margin-top: 110px;
+  padding: 160px 160px;
+  background-color: #fcda2e !important;
+  height: 1100px;
+  overflow: auto;
+}	
+.contact-form h1 {
+  font-size: 42px;
+  font-family: 'Pacifico', sans-serif;
+  margin: 0 0 50px;
   text-align: center;
-  font-size: 24px;
+}
+.contact-form .form-group {
+  margin-bottom: 20px;
+  padding: 0px 20px;
+}
+.contact-form .form-control, .contact-form {
+  min-height: 38px;
+  border-radius: 2px;
+  width: 100%;
+  display: block;
+}
+
+.btn {
+  min-height: 38px;
+  border-radius: 2px;
+  align-self: center;
+}
+.button-center {
+  text-align-last: center;
+}
+.contact-form .form-control {
+	border-color: #e2c705;
+}
+.contact-form .form-control:focus {
+	border-color: #d8b012;
+	box-shadow: 0 0 8px #dcae10;
+}
+.contact-form .btn-primary {
+  width: 50%;
+  color: #fcda2e;
+  background: #000;
+  margin-top: 20px;
+  border: none;
+}
+.contact-form .btn-primary:hover {
+  color: #fff; 
+}
+.contact-form .btn-primary i {
+  margin-right: 5px;
+}
+
+.contact-form textarea {
+  height: 14vh;
+  resize: vertical;
+}
+.bs-example {
+  margin: 20px;
+}
+
+.col-50 {
+  float: left;
+  width: 50%;
+  margin-top: 6px;
 }
 
 /* Responsive layout - when the screen is less than 600px wide, make the two columns stack on top of each other instead of next to each other */
 @media screen and (max-width: 600px) {
-  .contact-container {
+  .col-50, input[type=submit] {
     width: 100%;
-    flex-direction: column;
+    margin-top: 10px;
+  }
+  .contact-form {
+    padding: 50px 10px;
+  }
+  form {
+    overflow: hidden;
+    padding: 5px;
+    width: 350px; 
+    margin: auto;
+    align-content: center;
   }
+
+}
+
+/* Style for back to top button */
```

Comment:
I like the comments here. Some folks might think they are overkill but, IMO, it's nice to see what you want it to look like!